test/classes/initializers/dflt-ctor-generic-uninit-field-const-err-1.good

```diff
@@ -0,0 +1,2 @@
+dflt-ctor-generic-uninit-field-const-err-1.chpl:7: error: unresolved call 'A.init()'
+dflt-ctor-generic-uninit-field-const-err-1.chpl:3: note: candidates are: A.init(c)
```

Comment:
Yes (and I think I have a test of that).  We require an argument for the constructor of a generic type.